modules/core.mjs

```diff
@@ -49,10 +49,16 @@ function setBatchMode(on) { batch_mode = !!on; }
 /** @summary Indicates if running inside Node.js */
 function isNodeJs() { return nodejs; }
 
+/** @summary Dynamically import module */
+async function dynamicImport(moduleName) {
+   /* exclude_from_build */ if (isNodeJs()) { return await import(moduleName); }
+   return Promise.resolve(undefined);
+}
+
```

Comment:
I guess, one can avoid `async` and `await` here.
I try do not use such construct - while it has side effects during SVG rendering.

Reply:
It's good practice to mark functions that return promises as `async`.

<https://typescript-eslint.io/rules/promise-function-async/>

Reply:
I know, but web browser makes extra HTML/SVG rendering when calling such function.
Therefore none of function in JSROOT should use such signature.
When used in user code - fine, while than it not affect rendering performance.